### PR TITLE
Added shortcodes for maistraBranch and maistraVersion

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: topics/docs/examples/bookinfo.adoc [hunk 1]
```diff
@@ -63,67 +69,86 @@ deployment.extensions/reviews-v2 created
...
 
 === Add default destination rules
  . If you did *not* enable mutual TLS:
+
 +
-```sh
+
+[source, bash]
+----
 $ oc -n ${BOOKINFO_NS} apply -f https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/destination-rule-all.yaml
```
Comment (jwendell) [inline topics/docs/examples/bookinfo.adoc]: missing here

File: topics/docs/examples/bookinfo.adoc [hunk 2]
```diff
@@ -63,67 +69,86 @@ deployment.extensions/reviews-v2 created
...
-```
+----
  . If you *did* enable mutual TLS:
+
 +
-```sh
+
+[source, bash]
+----
 $ oc -n ${BOOKINFO_NS} apply -f https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/destination-rule-all-mtls.yaml
```
Comment (jwendell) [inline topics/docs/examples/bookinfo.adoc]: and here

**Review by jwendell — CHANGES_REQUESTED** [review]
it rendered as 1.2, is that right?

File: topics/docs/examples/bookinfo.adoc (hunk 1 above)
Comment (brian-avery) [reply topics/docs/examples/bookinfo.adoc]: That's a link to Istio, not to Maistra. I think Marko's PR addresses that one.

File: topics/docs/examples/bookinfo.adoc (hunk 2 above)
Comment (brian-avery) [reply topics/docs/examples/bookinfo.adoc]: That's a link to Istio, not to Maistra. I think Marko's PR addresses that one.

**Comment by brian-avery** [comment]
Yep. This PR is against the Maistra-1.2 branch. When I cherry-pick to Maistra 1.1, it will pick up the data from release.yaml there and render as Maistra-1.1

File: topics/docs/examples/bookinfo.adoc (hunk 1 above)
Comment (brian-avery) [reply topics/docs/examples/bookinfo.adoc]: Marko's PR merged. Updated with changes from that.

File: topics/docs/examples/bookinfo.adoc (hunk 2 above)
Comment (brian-avery) [reply topics/docs/examples/bookinfo.adoc]: Marko's PR merged. Updated with changes from that.

**Review by luksa — APPROVED** [review]

**Review by jwendell — APPROVED** [review]

**Pull request merged**